pkgmgr/registry.go

```diff
@@ -91,6 +91,37 @@ docker run --rm -ti ghcr.io/blinklabs-io/mithril-client:0.7.0-1 $@
 			},
 		},
 	},
+	{
+		Name:        "ogmios",
+		Version:     "v5.6.0-mainnet",
```

Comment:
We'll need 6.0+ for compatibility with our given cardano-node versions.

I suggest `v6.1.0`, instead.